README.md

```diff
@@ -1,83 +1,176 @@
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
 
-This GitHub Action is going to run [Rubocop](https://docs.rubocop.org/en/stable/) and [Stylelint](https://stylelint.io/) to help you find style issues.
+<!-- PROJECT DESCRIPTION -->
 
-[Stylelint](https://stylelint.io/) is a linter for your stylesheets that helps you avoid errors and enforce conventions.
+# 📖 [Budget App] <a name="about-project"></a>
 
-[Rubocop](https://docs.rubocop.org/en/stable/) is a Ruby static code analyzer (a.k.a. linter) and code formatter. It will enforce many of the guidelines outlined in the community [Ruby Style Guide](https://rubystyle.guide/).
+💰💼 GDM Budget_smart is a mobile web application where you can manage your budget: you have a list of transactions associated with a category, so that you can see how much money you spent and on what.
 
-Please do the following **steps in this order**:
+## 🛠 Built With <a name="built-with"></a>
 
-1. In the first commit of your feature branch create a `.github/workflows` folder and add a copy of [`.github/workflows/linters.yml`](.github/workflows/linters.yml) to that folder.
-    - **Remember** to use the file linked above
-    - **Remember** that `.github` folder starts with a dot.
-2. **Do not make any changes in config files - they represent style guidelines that you share with your team - which is a group of all Microverse students.**
-    - If you think that change is necessary - open a [Pull Request in this repository](../README.md#contributing) and let your code reviewer know about it.
-3. When you open your first pull request you should see the result of the GitHub Actions:
+### Tech Stack <a name="tech-stack"></a>
 
-![gh actions checks](../assets/images/gh-actions-rubocop-stylelint-checks.png)
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">Rails</a></li>
+  </ul>
+</details>
 
-Click on the `Details` link to see the full output and the errors that need to be fixed:
+<!-- Features -->
 
-![gh actions failing checks](../assets/images/gh-actions-rubocop-failing-checks.png)
+### Key Features <a name="key-features"></a>
 
-## Set-up linters in your local env
+- Gitflow execution
+- Register an account
+- Login registered account
+- Create categories and transactions
+- Upload Profile photo
 
-**Note**: The `npm` package manager is going to create a `node_modules` directory to install all of your dependencies. You shouldn't commit that directory. To avoid that, you can create a [`.gitignore`](https://git-scm.com/docs/gitignore) file and add `node_modules` to it:
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
+### 🚀Live Demo <a name="Live Demo"></a>
+
+[Live](https://gdm-budget-smart2.onrender.com)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+### Presentation <a name="Presentation"></a>
+
+[presentation](https://drive.google.com/file/d/1WKvpQqHMQDH3GazsHVhR_IJ0Q8omJMJ6/view?usp=sharing)
+
```

Comment:
- [ ] While it seems that you did make the changes as requested by the first reviewer @cynthiainga . The project that is being presented seems to be different from the one that is currently in the project.

**This is the current project landing page:**

![image](https://github.com/Gardimy/Budget_app/assets/86318284/57dc238f-c429-4c43-bb9d-bed9b5691aea)

**This is the project that is presented in the video:**

![image](https://github.com/Gardimy/Budget_app/assets/86318284/a8d51229-ed6f-4462-8673-1ed08eec87dd)

It would be nice if you took the time to record the video of the correct project.